tablerendering/tablerendering.go

```diff
@@ -0,0 +1,35 @@
+package tablerendering
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/rodaine/table"
+)
+
+// RenderTable provides rendering of a table on cli
```

Comment:
Maybe change doc string to something more precise. RenderTable prints a table to the given io.Writer.